src/components/community-link/community-link.js

```diff
@@ -0,0 +1,81 @@
+import '@brightspace-ui/core/components/link/link.js';
+import '@brightspace-ui/core/components/button/button-icon.js';
```

Comment:
This is the wrong import.
```suggestion
import '@brightspace-ui/core/components/button/button-subtle.js';
```